packages/core/src/rules/oas3/no-invalid-media-type-examples.ts

```diff
@@ -35,6 +35,9 @@ export const ValidContentExamples: Oas3Rule = (opts) => {
             location = isMultiple ? resolved.location.child('value') : resolved.location;
             example = resolved.node;
           }
+          if (Object.prototype.hasOwnProperty.call(example, 'externalValue')) {
```

Comment:
Hi @xeptore. Thanks for bringing this up! 

When both `value` and `externalValue` are present, the proposed solution does not validate the `value` at all. Theoretically, someone could disable the `no-example-value-and-externalValue` rule and that would prevent the wrong value from being caught. 

My suggestion is to check if the `value` actually exists, and if so, validate it. Otherwise, just skip the validation:

```suggestion
          if (isMultiple && typeof example.value === 'undefined') {
```

Does that make sense?

Reply:
Sure, committed in 728ebac2168c9004093de27cde2909026e9adc3e.